fmt/src/visit.rs

```diff
@@ -318,6 +318,78 @@ pub trait Visitor {
 
         Ok(())
     }
+
+    fn visit_yul_block(
+        &mut self,
+        loc: Loc,
+        _stmts: &mut Vec<YulStatement>,
+        _attempt_single_line: bool,
+    ) -> Result<(), Self::Error> {
+        self.visit_source(loc)
+    }
+
+    fn visit_yul_expr(&mut self, expr: &mut YulExpression) -> Result<(), Self::Error> {
+        self.visit_source(expr.loc())
+    }
+
+    fn visit_yul_assignment(
+        &mut self,
+        loc: Loc,
+        _exprs: &mut Vec<YulExpression>,
+        _expr: &mut YulExpression,
+    ) -> Result<(), Self::Error> {
+        self.visit_source(loc)
+    }
+
+    fn visit_yul_break(&mut self, loc: Loc) -> Result<(), Self::Error> {
```

Comment:
since this does the same thing, I think we can just use `visit_break` and `visit_continue`